README.md

```diff
@@ -88,7 +88,7 @@ Together, we can make Unturned.Info **better**!
 
 First off, thanks for taking the time to contribute! Contributions are what make the open-source community such an amazing place to learn, inspire, and create. Any contributions you make will benefit everybody else and are **greatly appreciated**.
 
-We have set up a separate document containing our [contribution guidelines](docs/CONTRIBUTING.md).
+We have set up a separate document containing our [contribution guidelines](https://github.com/Nonanti/unturned-3-knowledgebase/blob/master/contributing.md).
```

Comment:
It looks like you've accidently set this to link to your own repository instead of the unturned-info repo or a relative markdown link like `contributing.md` which would be ideal.

Reply:
You have to include it in your own repository

Example:
https://github.com/unturned-info/unturned-3-knowledgebase/blob/master/contributing.md

Reply:
I'm not sure what you're getting at?

Reply:
![image](https://user-images.githubusercontent.com/45326482/151517971-785b65da-9715-4e5f-9652-639b238a36ca.png)